### PR TITLE
[MDS-5811] gis_export_view migration

### PR DESCRIPTION
**Pull request by matbusby-fw**
## Objective 

[MDS-5811](https://bcmines.atlassian.net/browse/MDS-5811)

- Added mine_purpose, regional_contact and latest_response_date to the gis_export materialized view
- Manually added this migration to the flyway history for dev and test to prevent it running there
- Updated the gis model in the backend to accomodate the new fields

### CODE REVIEW

**Review by henryoforeh-dev — DISMISSED** [review]
Nice!

File: migrations/sql/V2024.04.02.17.05__update_now_application_gis_export_view.sql [hunk 1]
```diff
@@ -0,0 +1,451 @@
...
+        WHERE schemaname = 'public'
+          AND matviewname = 'now_application_gis_export_view'
+    )
+    THEN
+        -- If it is a materialized view
+        EXECUTE 'DROP MATERIALIZED VIEW IF EXISTS public.now_application_gis_export_view';
+    END IF;
+END $$;
+
+CREATE MATERIALIZED VIEW public.now_application_gis_export_view
```
Comment (asinn134) [inline migrations/sql/V2024.04.02.17.05__update_now_application_gis_export_view.sql]: Just want to make sure but is the only change in the big query just adding the new fields?

Comment (henryoforeh-dev) [reply migrations/sql/V2024.04.02.17.05__update_now_application_gis_export_view.sql]: Yes. Just adding the new columns as required.

**Review by henryoforeh-dev — DISMISSED** [review]

**Review by asinn134 — DISMISSED** [review]

**Review by henryoforeh-dev — APPROVED** [review]
Docker!

**Review by taraepp — APPROVED** [review]

**Pull request merged**